### PR TITLE
Fixed a few tex-o's...

### PR DESCRIPTION
**Pull request by jj5**
Hi there. A few small fixes and improvements...

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -1114,7 +1114,7 @@ \section{Existence of a solution}
 eliminate $h_n, k_m, h_i, k_j$ yb means of the normalizing equations. In this
 form, the complete system is as follows:
 
-\mathbf{Normalizing Equations}
+$\mathbf{Normalizing Equations}$
```
Comment (j2kun) [inline main.tex]:
```suggestion
\textbf{Normalizing Equations}
```

File: main.tex [hunk 2]
```diff
@@ -1123,8 +1123,8 @@ \section{Existence of a solution}
 k_j \int_{b_j}^{b_{j+1}} g^*(t) dt &= 1, \qquad j=1, \dots, m-1 \label{eqn:existence-17}
 \end{align}
 
-\mathbf{Equations from Lemma~\ref{lemma:coefficient-equations} and
-Lemma~\ref{lemma:optimality-of-corresponding-strategies}}
+$\mathbf{Equations from Lemma~\ref{lemma:coefficient-equations} and
```
Comment (j2kun) [inline main.tex]:
```suggestion
\textbf{Equations from Lemma~\ref{lemma:coefficient-equations} and
```

File: main.tex [hunk 3]
```diff
@@ -1123,8 +1123,8 @@ \section{Existence of a solution}
...
+Lemma~\ref{lemma:optimality-of-corresponding-strategies}}$
```
Comment (j2kun) [inline main.tex]:
```suggestion
Lemma~\ref{lemma:optimality-of-corresponding-strategies}}
```

File: main.tex (hunk 1 above)
Comment (j2kun) [reply main.tex]: Good catch! Note that `\textbf` is the canonical way to make paragraph text bold, so please include my little patch :)

**Review by j2kun — CHANGES_REQUESTED** [review]
Wow! I had no idea anyone was paying attention to this repo (and I still have to work through the paper myself for a sanity check).

I will eventually post this on the arXiv, so if you'd like I can include your name as a thanks at the end.

**Comment by jj5** [comment]
I'm happy for you to leave my name off. I'm still new to github so please bear with me as I figure out how pull requests work... :P

**Comment by jj5** [comment]
Oh. I closed the pull request. But I don't think I was supposed to do that. So now I've reopened the pull request... help? :P

**Review by j2kun — APPROVED** [review]

**Comment by j2kun** [comment]
Looks good to me! Merging, and thanks again!

**Pull request merged**